go/vt/servenv/version.go

```diff
@@ -1,10 +1,22 @@
-package servenv
+/*
+Copyright 2022 The Vitess Authors.
 
-import "flag"
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
 
-const versionName = "14.0.0-SNAPSHOT"
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// THIS FILE IS AUTO-GENERATED DURING NEW RELEASES
+// DO NOT EDIT
```

Comment:
This is cool :)
How does the generated file get committed? Does that require a manual step?

Reply:
It gets committed automatically yes